test/Microsoft.Sbom.Api.Tests/Config/Validators/ManifestInfoValidatorTests.cs

```diff
@@ -15,6 +17,11 @@ namespace Microsoft.Sbom.Api.Tests.Config.Validators;
 public class ManifestInfoValidatorTests
 {
     private readonly Mock<IAssemblyConfig> mockAssemblyConfig = new Mock<IAssemblyConfig>();
+    private readonly HashSet<ManifestInfo> supportedManifestInfosForTesting = new HashSet<ManifestInfo>
```

Comment:
There's also a TestManifestInfo provided in Constants.cs, should that be added to this list for consistency?